src/Controller/Api/GalleryController.php

```diff
@@ -520,6 +486,8 @@ protected function getMediaManager()
     /**
      * Write a Gallery, this method is used by both POST and PUT action methods.
      *
+     * @View(serializerGroups={"sonata_api_read"}, serializerEnableMaxDepthChecks=true)
```

Comment:
Wait, @kunicmarko20  because this method is protected and called from the public methods, that's probably why it was created like that, I think I need to revert this change

Reply:
I have decided to revert this, because it was used on a protected method, and will probably break BC that a method that returns a view, suddenly starts returning a media.

Let's not create any issue with this change. wdyt?